ui/lib/kv/addon/components/page/secret/edit.hbs

```diff
@@ -70,23 +70,21 @@
   </div>
   <div class="box is-fullwidth is-bottomless">
     <div class="control">
-      <button
+      <Hds::Button
+        @text="Save"
+        @icon={{if this.save.isRunning "loading"}}
         type="submit"
-        class="button is-primary {{if this.save.isRunning 'is-loading'}}"
         disabled={{this.save.isRunning}}
         data-test-kv-save
-      >
-        Save
-      </button>
-      <button
-        type="button"
-        class="button has-left-margin-s"
+      />
+      <Hds::Button
+        @text="Cancel"
+        @color="secondary"
+        class="has-left-margin-s"
         disabled={{this.save.isRunning}}
         {{on "click" this.onCancel}}
         data-test-kv-cancel
-      >
-        Cancel
-      </button>
+      />
```

Comment:
As discussed we are going to add `display: inline` as an override to the hds button class to avoid scope creep and the button set adoption will happen in as a follow up to this work.